app/controllers/users.js

```diff
@@ -22,23 +23,43 @@ class UsersController extends BaseController {
       inMemoryBlockList
     );
     this.sessionRepository = new SessionRepository(ACTIVE);
+    this.contactMatchRepository = new ContactsMatchRepository();
   }
 
   async create(ws, data) {
     const { id: requestId, user_create: reqData } = data;
 
     const isUserCreate = await User.findOne({ login: reqData.login });
```

Comment:
when we name a variable like this
```
const isUserCreate
```

it means it will hold a boolean.
But in our case it holds a User object.

Either to like this (transform to boolean):
```
const isUserCreated = !!(await User.findOne..)
```

or

```
const existingUser = await User.findOne..
```